InstallSpinnaker.sh

```diff
@@ -571,12 +584,10 @@ unzip -u -o -q packer_0.8.6_linux_amd64.zip -d /usr/bin
 popd
 rm -rf /tmp/packer
 
-if [[ "x$DEPENDENCIES_ONLY" != "x" ]]; then
-    exit 0
-fi
-
 ## Spinnaker
-install_spinnaker
+if [[ "x$DEPENDENCIES_ONLY" == "x" ]]; then
```

Comment:
I removed this gate and made the script where it did before since we aren't setting any google initializations.